test/flycheck-test.el

```diff
@@ -3316,9 +3316,18 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
                  "language/erlang/rebar3/_build/default/lib/dependency/_build")))))
 
 (flycheck-ert-def-checker-test eruby-erubis eruby nil
+  (let ((flycheck-disabled-checkers '(eruby-ruumba)))
   (flycheck-ert-should-syntax-check
```

Comment:
You want to re-indent the `let` body to pass the CI.

Reply:
Sorry, I think I have them properly formatted now.